libafl/src/executors/inprocess.rs

```diff
@@ -860,7 +869,7 @@ mod windows_exception_handler {
             LeaveCriticalSection(data.critical as *mut RTL_CRITICAL_SECTION);
             compiler_fence(Ordering::SeqCst);
 
-            windows_delete_timer_queue(x);
+            data.reset();
```

Comment:
Hi 👋 
Do you mean executor.reset() here?

Reply:
yea thanks and sorry for such obvious mistakes, I don't have a Windows test setup and was waiting for CI 😅

Reply:
ci is broken with the updated rust/clippy btw (on main)
Once it is fixed I can check if it builds on win